pkg/engine/validation.go

```diff
@@ -206,7 +206,7 @@ func (v *validator) validate() *response.RuleResponse {
 		return ruleError(v.rule, utils.Validation, "failed to evaluate preconditions", err)
 	}
 
-	if !preconditionsPassed {
+	if !preconditionsPassed && v.ctx.Policy.Spec.ValidationFailureAction != "audit" {
```

Comment:
We have update to use interface method

```suggestion
	if !preconditionsPassed && v.ctx.Policy.GetSpec().ValidationFailureAction != kyverno.Audit {
```